docs/spark-queries.md

```diff
@@ -270,17 +270,22 @@ order by made_current_at
 
 ### Files
 
-To show a table's current data files:
+To show a table's current files:
 
 ```sql
 SELECT * FROM prod.db.table.files;
 ```
 
-|content|file_path                                                                                                                                   |file_format|spec_id|partition|record_count|file_size_in_bytes|column_sizes      |value_counts    |null_value_counts|nan_value_counts|lower_bounds           |upper_bounds           |key_metadata|split_offsets|equality_ids|sort_order_id|
+| content | file_path | file_format | spec_id | partition | record_count | file_size_in_bytes | column_sizes | value_counts | null_value_counts | nan_value_counts | lower_bounds | upper_bounds | key_metadata | split_offsets | equality_ids | sort_order_id | readable_metrics |
 | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- |
```

Comment:
alignment looks off, I think you want to match columns count here as well

<img width="566" alt="image" src="https://github.com/apache/iceberg/assets/5961173/013c9415-34c6-4d50-85b2-aab2d0f8f9be">

Reply:
FIxed, please check again.